cpython-unix/build.py

```diff
@@ -186,6 +186,8 @@ def add_target_env(env, build_platform, target_triple, build_env):
         if not os.path.exists(sdk_path):
             raise Exception("macOS SDK path %s does not exist" % sdk_path)
 
+        env["APPLE_SDK_PATH"] = sdk_path
```

Comment:
What is this for?

Reply:
To pass through to `xcode_path = os.getenv("APPLE_SDK_PATH")`?

Reply:
Yeah that's right.